### PR TITLE
Updated rake task functionality for repo merge

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by rsalevsky** [comment]
+1

**Comment by MaximilianMeister** [comment]
+1

File: config/barclamps.yml [hunk 1]
```diff
@@ -1,33 +1,5 @@
...
-  - openstack
-  - pacemaker
-  - rabbitmq
-  - swift
-  - tempest
-  - trove
+- crowbar-core
+- crowbar-openstack
+- crowbar-ceph
+- corbwar-hyperv
```
Comment (aspiers) [inline config/barclamps.yml]: Typo here

Comment (tboerger) [inline config/barclamps.yml]: This is already fixed in an followup pull request. This pull request missed a fixing commit.

Comment (aspiers) [inline config/barclamps.yml]: Thanks, I didn't see that.

**Pull request merged**